jaxrs/src/main/java/com/cerner/beadledom/jaxrs/JaxRsParamConditions.java

```diff
@@ -43,10 +43,9 @@ private JaxRsParamConditions() {
   public static void checkParam(boolean expression, Object errorMessage) {
```

Comment:
It's `Object` because this is similar to the methods in the Guava class `Preconditions` so I stuck with a similar signature.